DOC-5229 enable percentiles for some DD metrics (#14709)
cockroachdb/docs: cockroachcloud/monitoring-page.md

Metric values and time-series graphs in Datadog are not guaranteed to match those in the [DB Console](#access-the-db-console), due to differences in how CockroachDB and Datadog calculate and display metrics.
{{site.data.alerts.end}}

#### Enable percentiles for selected metrics

> A subset of CockroachDB metrics require that you explicitly enable percentiles for them in the Datadog interface, or graphs that display data for them will fail to render properly.

Reviewer: ```suggestion
A subset of CockroachDB metrics require that you explicitly enable percentiles for them in the Datadog interface. Graphs that display data for these metrics will fail to render properly otherwise.
```

To enable percentiles for these metrics, perform the following steps:

> 1. In your Datadog interface, select **Metrics** then **Summary** from the sidebar.

Reviewer: I question the general practice of giving such detailed documentation for operations in another company's product's UI. It does sound like the setting is pretty buried, so I'm sure this level of detail would be helpful while it remains accurate, but Datadog could change those details out from under us, rendering the instructions more confusing than helpful. This is not a hard and fast objection, so feel free to keep it as is if you consider the benefits worth the cost in this particular case.
 
Checking the "distribution" checkbox seems like a useful and non-obvious pro tip, so I would suggest keeping that.

2. Check the **Distributions** checkbox in the **Metric Type** section to limit returned metrics to the subset of CockroachDB metrics that require percentiles support.

> 3. Select the metric you wish to enable percentiles for, expand its **Advanced** section, and click the **Edit** button. Note the text explaining the benefit of enabling percentiles.

Reviewer: ```suggestion
3. For each metric requiring percentiles:
    1. Expand its **Advanced** section
    1. Click the **Edit** button.
    1. Click the slider labeled **Enable percentiles and threshold queries**.
    1. Click the **Save** button.
```

> 4. Click the slider labeled **Enable percentiles and threshold queries**, then click the **Save** button.

Reviewer: ```suggestion
```

> 5. Repeat for the other metrics you wish to use.

Reviewer: ```suggestion
```

> You only need to perform this once per metric. Datadog graphs reliant on these metrics will begin rendering immediately once configured in this manner. 

Reviewer: ```suggestion
```
Both points seem to me like default expectations so perhaps unnecessary? I leave it to you, it's not a strong opinion.

> Only data receieved for these metrics once percentiles are enabled can be displayed; any data collected between enabling the Datadog integration and the enablement of percentiles for these specific metrics cannot be rendered.

Reviewer: ```suggestion
Only data received for these metrics once percentiles are enabled can be displayed; previously collected data cannot be rendered.
```
